src/comp.py

```diff
@@ -23,49 +25,94 @@ def __repr__(self):
 # whose name starts with 'D':
 
 print("Starts with D:")
-r = []  # TODO
+r = []
+for x in humans:
```

Comment:
For `comp.py`, we have to write a *list comprehension* for each of the exercises.

Here's a pretty comprehensive guide to list comprehension: https://www.datacamp.com/community/tutorials/python-list-comprehension